.github/workflows/integration-test-smoke-suite.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: TestCephSmokeSuite
       run: |
        export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-       SKIP_CLEANUP_POLICY=false go test -v -timeout 1800s -run CephSmokeSuite github.com/rook/rook/tests/integration
+       SKIP_CLEANUP_POLICY=false go test -v -timeout 1800s -failfast -run CephSmokeSuite github.com/rook/rook/tests/integration
```

Comment:
Does this mean any test assert will fail the test? We probably want to revert this, right?

Reply:
Yes. Why not keeping it? What's the point of continuing if the test fails? Gathering logs? I think these days it's quicker to jump in the runner and look at the logs ourselfve. What do you think?

Reply:
Does the test stop on `assert.Fail()` or only on `require.Fail()`? I would expect the tests to continue on `assert.Fail()` since those should not be fatal and we should see if there are other failures in the same run. How about if we instead change some asserts to requires?

Reply:
The problem with the suite is that it runs multiple tests all prefixed with `TestFoo`. What I observed is that no matter how you stop/fail (even `panic`), `go test` will run the next `TestBar`, then present the failure at the end.

The downside with failfast is that we will get errors one by one and we will fix them one by one. I still think it's a good idea so we are forced to always fix any issue (even if intermittent) that arises.